course-lessons/python-fundamentals/python-fundamentals-variables.md

```diff
@@ -0,0 +1,200 @@
+
+### Importance of Clear and Distinct Variable Names
+
+Just like you want to use good naming conventions for directories on your computer, you also want to assign clear and short names to variables, avoiding spaces or complicated wording, but still specific enough that you know what it is. Underscores (e.g. `boulder_precip_in`) can be helpful to add more detail to the name (e.g. location, measurement, units) and still preserve code readibility. 
+
+A key characteristic of variables is that once a variable is created within your code (e.g. **Jupyter Notebook** file), the value of that variable will remain the same and can be reused throughout the code.
+
+However, you can overwrite an existing variable if you create a new variable with the same name. 
+
+While there are occassions in which you might want to overwrite an existing variable to assign a new value to it, you want to make sure that you give variables both clear and distinct names to avoid *accidentally* overwriting variable values.  
+
+
+## Numeric Variables
+
+In **Python**, you can create variables to store numeric values such as  integers (`int`) (i.e. whole numbers) and floating point numbers (`float`) (i.e. decimal values).  
+
+For more advanced math applications, you can also use variables to work with <a href="https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex" target="_blank">complex numbers (see Python documentation for more details)</a>. 
+
+As described previously, you do not need to define which numeric type you want to use to create a variable. 
+
+For example, you can create a `int` variable called `boulder_precip_in`, which contains the rounded up value for the average annual precipitation in inches (in) in Boulder, Colorado. 
+<!-- #endregion -->
+
+```python
+boulder_precip_in = 21
+
+boulder_precip_in
+```
+
+You could also create a `float` variable to capture the data using decimal units. 
+
+```python
+boulder_precip_in = 20.68
+
+boulder_precip_in
+```
+
+Notice that in these examples, `boulder_precip_in` was overwritten by the second variable definition because both definitions assigned the same name to the variable. 
+
+
+## Text String Variables
+
+To create a variable containing a text string (`str`), you need to use quotations (`""`) around the value to identify the value as a text string (e.g. `variable_name = "text"`). 
+
+While in **Python** the single quote (`''`) and the double quote (`""`) are used interchangeably (see the official <a href="https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str" target="_blank">Python Docs</a> for more examples), it is good to choose one option and use it consistently. This textbook uses the double quote (`""`) for identifying text strings (`str`). 
```

Comment:
Minor change here: "This textbook uses" should be "The exercises and materials that you will explore use..."